lithops/config.py

```diff
@@ -226,7 +226,8 @@ def default_storage_config(config_file=None, config_data=None, backend=None):
 
 def extract_storage_config(config):
     s_config = {}
-    s_config['monitoring_interval'] = config['lithops']['monitoring_interval']
+    s_config['monitoring_interval'] = config['lithops'].get('monitoring_interval',
+                                                            LITHOPS_DEFAULT_CONFIG_KEYS['monitoring_interval'])
```

Comment:
You can use `c.LITHOPS_DEFAULT_CONFIG_KEYS` instead of importing the var again. 
And to avoid potential line length issues, it will probably look better like this:
```
    s_config['monitoring_interval'] = config['lithops'].get(
        'monitoring_interval', c.LITHOPS_DEFAULT_CONFIG_KEYS['monitoring_interval']
    )
```

Reply:
Of course. Changes applied. Thank you,